tools/gcp.py

```diff
@@ -68,19 +99,39 @@ def create_vm(self) -> None:
             f"count={self.config.accelerator_count}",
         ]
         self.print(" ".join(cmd))
-        subprocess.run(cmd)
+        p = subprocess.run(cmd)
 
-        # Agree to NVIDIA's prompt and install the GPU driver
+        # Agree to NVIDIA's prompt and install the GPU driver.
+        # This monster below is here bc the yes command
+        # and a gazillion alternatives do not work on circleci.
+        # reverse-engineered from /usr/bin/gcp-ngc-login.sh
```

Comment:
lol

Reply:
spent like 4 hours on this today!! 😠

Reply:
now you know how it works

Reply:
yea... what scares me is that there are so many things that could change here from one release to the other